pkg/controller/storagecluster/prometheus.go

```diff
@@ -105,8 +105,9 @@ func (r *ReconcileStorageCluster) CreateOrUpdatePrometheusRules(rule *monitoring
 			if err != nil {
 				return fmt.Errorf("failed while updating PrometheusRule: %v", err)
 			}
+		} else {
+			return fmt.Errorf("failed while creating PrometheusRule: %v", err)
```

Comment:
Why is this necessary?

Reply:
I think this looks good - otherwise it seems like this err will be returned every time the create fails with alreadyExists.

Reply:
Correct. We want to ignore `err = alreadyExists`.